user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -19,111 +19,295 @@ redirect_from:
+* **Debian 9 (stretch) template** : Install these templates if you wish to base some of your Qubes virtual machines on Debian instead of Fedora. 
+* **Whonix** : Install Whonix templates if you wish some of your qubes to be based on Whonix. Whonix let you route your entire network traffic trough Tor if you see fit. For more information about Whonix, have a look at their [website](https://www.whonix.org/). Depending on your threat model, you may need to install Whonix templates right away.
 
+Note that you will also be able to install  Add-Ons after the installation is completed. If you wish for your system to be more lightweight, 
+do not hesitate to un-check those options. 
 
-Installing to a USB drive
--------------------------
+<div class="alert alert-info" role="alert">
+    <i class="fa fa-question-circle"></i>
+    <b> Note : </b> By default, Qubes OS comes preinstalled with the lightweight Xfce desktop environnement for dom0, the main domain. Other desktop environments will be available to you after the installation is completed, although they may not be officially supported. 
+</div>
 
-Installing an operating system onto a USB drive can be a convenient and secure
-method of ensuring that your data is protected. Be advised that a minimum
-storage of 32 GB is required on the USB drive. This installation process may
-take longer than an installation on a standard hard disk. The installation
-process is identical to using a hard disk in conjunction with two exceptions:
+Click on "Done" as soon as you have made your choice to go back to Installation summary screen. 
 
-* Select the USB as the storage location for the OS. 
+<h3>Installation destination</h3>
 
-* Leave the option checked to “Automatically configure my Qubes installation to
-the disk(s) I selected and return me to the main menu”.
+Under the System section, you need to pick the installation destination. For this step to be completed, you need to select which storage device you would like your system to be installed on. By default, only local storage devices are shown. Under the Device Selection section, make sure that you select the correct installation destination. Ensure that your your target destination has a least 32 GiB of free space available.   
 
+For this setup, options will be left unchanged. By default, Qubes OS will partition the system itself using LUKS encryption on top of LVM, and will claim the entire storage device.
 
-Upgrading
----------
+<img src="/attachment/wiki/InstallationGuide/select-storage-device.png">
 
-For instructions in upgrading an existing installation, please see [Upgrade Guides].
+<div class="alert alert-danger" role="alert">
+    <i class="fa fa-exclamation-circle"></i>
+    <b>Attention : </b> Any data on the target storage device will be destroyed during the installation process, including any available partitions, so make your selection carefully. 
```

Comment:
There is a separate prompt for that ("I would like to make additional space available"), followed by a separate confirmation dialog where you select what partitions should be removed. Unless you select that, nothing is removed.

Reply:
I have modified the note